src/components/ActionButton.tsx

```diff
@@ -50,6 +50,7 @@ const actionCss = css`
 export const Overlay = styled(Row)<{ hasAction: boolean }>`
   border-radius: ${({ theme }) => theme.borderRadius}em;
   flex-direction: row-reverse;
+  margin-top: 12px;
```

Comment:
The button itself should not have a margin-top. It should be a part of the containing element (eg as a `gap`).

Reply:
<img width="641" alt="image" src="https://user-images.githubusercontent.com/5773490/194957723-44fdce8c-746a-436a-baf2-1788cb0fa310.png">
Where would you put the 12px here? (This currently puts in the `Row` wrapper for `SwapActionButton`)